### PR TITLE
Added profile for user, fix views

### PR DESCRIPTION
**Pull request by navvyua**

### CODE REVIEW

File: app/views/news/index.html.slim [hunk 1]
```diff
@@ -4,8 +4,9 @@ section.content
       .box
         .box-header.with-border
           h3.box-title News
-          - if current_user.admin?
-            button.btn.btn-info.btn-sm.pull-right data-target='#newsModal' data-toggle='modal' type='button' Add news
+          - if current_user && current_user.admin?
```
Comment (IlyaUmanets) [inline app/views/news/index.html.slim]: create new helper `current_admin?`

File: app/views/news/index.html.slim [hunk 2]
```diff
@@ -17,7 +18,8 @@ section.content
                   = link_to news_path(n) do 
                     strong = n.title
                   p = slice_news_text(n.text) 
-                  .text-muted.pull-right Created at - #{news_date_format(n.created_at)}
+                  .text-muted.pull-right
+                    | Created at #{news_date_format(n.created_at)} by #{link_to n.user.name, profile_user_path(n.user)}
```
Comment (IlyaUmanets) [inline app/views/news/index.html.slim]: Create helper for this.

File: app/views/users/show.html.slim [hunk 3]
```diff
@@ -4,45 +4,48 @@ section.content-header
...
+            = link_to 'Edit profile', nil, class: 'btn btn-primary btn-block btn-flat'
       .box.box-primary
         .box-header.with-border
-          h3.box-title About me
-          
         .box-body
           ul.list-group.list-group-unbordered
             li.list-group-item
               b First name
+              span.pull-right = @user.first_name || '-'
```
Comment (IlyaUmanets) [inline app/views/users/show.html.slim]: create helper for this
Example
```ruby
def format_user_data(field)
  field ? field : '-'
end
```

File: app/views/news/index.html.slim (hunk 1 above)
Comment (navvyua) [inline app/views/news/index.html.slim]: Created **current_user_admin?** helper.

File: app/views/news/index.html.slim (hunk 2 above)
Comment (navvyua) [inline app/views/news/index.html.slim]: Created **news_created_label**.

File: app/views/users/show.html.slim (hunk 3 above)
Comment (navvyua) [inline app/views/users/show.html.slim]: Done.

**Pull request merged**